pkg/autodiscover/autodiscover_operators.go

```diff
@@ -26,9 +26,26 @@ import (
 	"github.com/test-network-function/cnf-certification-test/pkg/configuration"
 	"helm.sh/helm/v3/pkg/release"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
 	"k8s.io/client-go/rest"
 )
 
+const (
+	istio = "istio-system"
+)
+
+func findnamespace(oc corev1client.CoreV1Interface) bool {
+	nsList, err := oc.Namespaces().List(context.TODO(), metav1.ListOptions{})
```

Comment:
What about verifying the installed-state  object as described in https://istio.io/latest/docs/setup/install/istioctl/? See below: 
Check what’s installed
The istioctl command saves the IstioOperator CR that was used to install Istio in a copy of the CR named installed-state. Instead of inspecting the deployments, pods, services and other resources that were installed by Istio, for example:

$ kubectl -n istio-system get deploy
NAME                   READY   UP-TO-DATE   AVAILABLE   AGE
istio-egressgateway    1/1     1            1           25s
istio-ingressgateway   1/1     1            1           24s
istiod                 1/1     1            1           20s

You can inspect the installed-state CR, to see what is installed in the cluster, as well as all custom settings. For example, dump its content into a YAML file using the following command:

$ kubectl -n istio-system get IstioOperator installed-state -o yaml > installed-state.yaml

The installed-state CR is also used to perform checks in some istioctl commands and should therefore not be removed.